packages/uploadthing/src/sdk/index.ts

```diff
@@ -2,7 +2,17 @@ import { generateUploadThingURL } from "@uploadthing/shared";
 
 import { UPLOADTHING_VERSION } from "../constants";
 
-const UT_SECRET = process.env.UPLOADTHING_SECRET;
+function guardServerOnly() {
+  if (typeof window !== "undefined") {
+    throw new Error("The `utapi` should only be used on the client.");
```

Comment:
"The `utapi` can only be used on the server."

Reply:
Whooops 😅